Source/EasyNetQ.Tests/Internals/AsyncQueueTests.cs

```diff
@@ -0,0 +1,43 @@
+using System.Threading;
+using System.Threading.Tasks;
+using EasyNetQ.Internals;
+using FluentAssertions;
+using Xunit;
+
+namespace EasyNetQ.Tests.Internals
+{
+    public class AsyncQueueTests
+    {
+        [Fact]
+        public async Task Should_enqueue_dequeue()
+        {
+            using var queue = new AsyncQueue<int>();
+            queue.Enqueue(1);
+            queue.Count.Should().Be(1);
+            var dequeueTask = queue.DequeueAsync();
+            (await dequeueTask.ConfigureAwait(false)).Should().Be(1);
```

Comment:
I suggest you to remove all `ConfigureAwait(false)` from tests. It is useless in test projects where is no syncronization context.

Reply:
Will do it a bit later in separate PR.